app/parse.py

```diff
@@ -8,8 +14,51 @@
     tags: list[str]
 
 
+QUOTE_ROWS = [field.name for field in fields(Quote)]
+
+
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one("span.text").text,
+        author=quote_soup.select_one("small.author").text,
+        tags=[tag.get_text() for tag in quote_soup.select(".tags > a.tag")],
+    )
+
+
+def get_quotes_from_single_page(soup: BeautifulSoup) -> list[Quote]:
+    quotes = []
+
+    products = soup.select(".quote")
+    for product in products:
+        quote = parse_single_quote(product)
+        quotes.append(quote)
+    return quotes
+
+
+def parse_pagination() -> list[Quote]:
+    results = []
+    counter = 1
+    while requests.get(f"{BASE_URL}/page/{counter}/").status_code == 200:
+        print(f"parsed page #{counter}")
+        page = requests.get(f"{BASE_URL}/page/{counter}/").content
```

Comment:
Similar to line 41, this line makes an HTTP request without handling exceptions. It's important to handle potential exceptions to avoid runtime errors.